src/main/java/uk/gov/register/core/Entry.java

```diff
@@ -48,11 +48,12 @@ public Instant getTimestamp() {
 
     @SuppressWarnings("unused, used from DAO")
     @JsonProperty("item-hash")
+    @JsonIgnore
```

Comment:
Is it only the RDA who are consuming this property? Do we want to make a breaking change or should this be supported alongside the new `item-hashes` for some transitional period?